convert/convert.go

```diff
@@ -28,10 +29,22 @@ func run(path string, overwrite bool) error {
 	if err != nil {
 		return err
 	}
+
+	// Look for a .gunkconfig
+	absPath, _ := filepath.Abs(path)
+	cfg, err := config.Load(filepath.Dir(absPath))
```

Comment:
I think this would be simpler if you just did `Load("")`, which will use $PWD. We do use gunk packages as the base for finding gunkconfig files, but I think that makes less sense when we're dealing with single protobuf files.

Reply:
Doing `Load("")` will work only if you are executing gunk from the same root directory than your gunkconfig. 
For instance, I'm doing `go build && ./gunk convert ../../examples/pb/util.proto --overwrite`. Like that I'm covering both cases.

Reply:
Hmm, `Load("")` simply looks for the config starting at the current directory, and then walking up the chain of parent directories until it finds the root of the project (a gunk config, or a git directory, etc), or the root of the filesystem. Isn't that enough?